src/layout/Modal.js

```diff
@@ -25,10 +25,11 @@ let Modal = ({ isOpen, children, style = {} }) => (
           style={{
             backgroundColor: '#fff',
             borderRadius: 5,
-            maxWidth: 'fit-content',
+            maxWidth: 640,
+            overflowX: 'scroll',
             // minHeight: 300,
             margin: '0 auto',
-            padding: 30,
```

Comment:
changing from padding to border as this means the scrollable x overflow won't overlap the visible whitespace